app/src/main/res/values/strings.xml

```diff
@@ -41,4 +49,15 @@
     <string name="ok">OK</string>
     <string name="registration_error">Registration Error</string>
     <string name="network_error">Sorry, an error occurred registering your account, please try again.</string>
+    <string name="shop_website">Shop website</string>
+    <string name="shop_phone_number">"Phone: "</string>
+    <string name="shop_phone_number_not_specified">Phone: not specified</string>
+    <string name="shop_address">"Address: "</string>
+    <string name="google_maps_search_url"><![CDATA[https://www.google.com/maps/search/?api=1&query=%1$s]]></string>
```

Comment:
Please move  google_maps_search_url to local.properties

Reply:
Moved